src/nvim/CMakeLists.txt

```
@@ -679,6 +679,14 @@ if(UNIX)
   endif()
 endif()
 
+if(NOT LUAJIT_FOUND)
+  message(STATUS "luajit not found, skipping unit tests")
+else()
+  glob_wrapper(UNIT_TEST_FIXTURES ${PROJECT_SOURCE_DIR}/test/unit/fixtures/*.c)
+  list(APPEND NVIM_SOURCES ${UNIT_TEST_FIXTURES})
+  target_compile_definitions(main_lib INTERFACE UNIT_TESTING) # BULL
```

Minor nitpick: if we want this to only apply to nvim and not libnvim
```suggestion
  target_sources(nvim PRIVATE ${UNIT_TEST_FIXTURES})
  target_compile_definitions(nvim PRIVATE UNIT_TESTING) # BULL
```

---

This would need to be moved down after the nvim target is defined in that case.